### PR TITLE
chore: update v11 branch with latest from main branch

### PR DESCRIPTION
**Pull request by alisonjoseph**
Sync main into v11 branch 🙏

### CODE REVIEW

**Review by joshblack — APPROVED** [review]

**Comment by esmahh** [comment]
I have read the DCO document and I hereby sign the DCO.

**Comment by aledavila** [comment]
recheck

**Review by tay1orjones — APPROVED** [review]
The new notification docs and the notification migration guidance are intact as expected

File: src/pages/guidelines/color/overview.mdx [hunk 1]
```diff
@@ -250,7 +250,10 @@ In the dark themes, layers become one step lighter with each added layer.
 
 <Tab label="Gray 90">
 
-![A dashboard in the Gray 90 theme](images/color-overview-themes-gray90.png)
+Gray 90 components are paired with Gray 100 backgrounds to make the Gray 100
```
Comment (aagonzales) [inline src/pages/guidelines/color/overview.mdx:253]: This is the old content being imported here. We need to keep the image instead.

File: src/pages/guidelines/color/overview.mdx [hunk 2]
```diff
@@ -250,7 +250,10 @@ In the dark themes, layers become one step lighter with each added layer.
...
+Theme and Gray 80 components with Gray 90 backgrounds to make the Gray 90 Theme.
+Some components, such as buttons and toggles, share the same colors across both
+dark themes.
```
Comment (aagonzales) [inline src/pages/guidelines/color/overview.mdx:256]:
```suggestion
![A dashboard in the Gray 90 theme](images/color-overview-themes-gray90.png)
```

**Review by aagonzales — CHANGES_REQUESTED** [review]
Just one spot where something didn't migrated old content.

File: src/pages/guidelines/color/overview.mdx (hunk 1 above)
Comment (alisonjoseph) [reply src/pages/guidelines/color/overview.mdx]: updated!

**Review by aagonzales — APPROVED** [review]

**Review by aledavila — APPROVED** [review]

**Pull request merged**